sdktests/server_side_data_system_base.go

```diff
@@ -0,0 +1,112 @@
+package sdktests
+
+import (
+	"time"
+
+	"github.com/launchdarkly/go-sdk-common/v3/ldcontext"
+	"github.com/launchdarkly/go-sdk-common/v3/ldvalue"
+	"github.com/launchdarkly/go-server-sdk-evaluation/v3/ldbuilders"
+	"github.com/launchdarkly/sdk-test-harness/v2/framework/ldtest"
+	"github.com/launchdarkly/sdk-test-harness/v2/servicedef"
+	"github.com/redis/go-redis/v9"
+	"github.com/stretchr/testify/require"
+)
+
+func doServerSideDataSystemTests(t *ldtest.T) {
+	rdb := redis.NewClient(&redis.Options{
```

Comment:
A couple things to note. As this work currently stands, this only supports redis. Since this work is being merged into a separate feature branch, I'm taking a very iterative approach here.

In this first pass, we just expect that you have redis up and running at this host and port. As the tests expand, that will eventually include work to support dynamodb and consul. At that point, we will probably start refactoring out into some docker container stuff.

All that to say, I know this isn't ideal, but a lot of this is a stepping stone.